cmd/plugins/topology-aware/policy/pod-preferences.go

```diff
@@ -381,52 +386,68 @@ func cpuAllocationPreferences(pod cache.Pod, container cache.Container) (int, in
 	qosClass := pod.GetQOSClass()
 	fraction := int(request.MilliValue())
 	prio := defaultPrio // ignored for fractional allocations
+	limit := 0
+
+	switch qosClass {
+	case corev1.PodQOSBestEffort:
+	case corev1.PodQOSBurstable:
+		if lim, ok := reqs.Limits[corev1.ResourceCPU]; ok {
+			limit = int(lim.MilliValue())
+		} else {
+			limit = unlimitedCPU
```

Comment:
This suggests that limit for BestEffort pods is unlimited, but on line 417 it seems that cpuAllocationPreferences() returns limit==0 for BestEfforts anyway. If this is the case, perhaps it would be clearer to assign/keep limit=0 here.